enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -239,8 +240,11 @@ def main(args=None):
     parser.add_argument("--config-dir", help="configuration folder", nargs="+")
     parser.add_argument("--bind", help="KME listener url for requests to KME",
                         type=str)
-    parser.add_argument(
-        "--worker_id", help="Id of worker in plain text", type=str)
+    parser.add_argument("--worker_id",
+                        help="Id of worker in plain text", type=str)
+    parser.add_argument("--cleanup_work_orders",
+                        help="Comma-separated list of work orders to recover",
+                        type=str)
```

Comment:
What is the format of work order  to be passed as argument?. If it's hex string then it is not user friendly.  Also how will uses know which work orders to pass. Clean up all work orders may be a better option.

Reply:
This is an optional argument specifically usable when a situation is hit where we need to recover only selective work-orders.